assets/js/util/ip-cidr.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * IP & CIDR Utility Functions.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* eslint-disable no-bitwise */
```

Comment:
```suggestion
```

Please disable ESLint rules per-instance, preferably with a comment explaining why 🙂

Reply:
Works for me, fair enough 👍🏻